src/restc_body.erl

```diff
@@ -21,8 +21,8 @@ decode(_, <<>>, Opts) ->
 decode(<<"application/json">>, Body, Opts0) ->
     Opts =
         case lists:member(return_maps, Opts0) of
-            true -> [return_maps];
-            false -> []
+            true -> [{return_maps, true}];
```

Comment:
Just to make sure.
You are making this change because the `jsx` dependency was updated to a new major version?

Reply:
Yes, the behavior was changed to default to maps